doc/move_code/objects_tutorial/sources/Ch1.move

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module Tutorial::Ch1 {
```

Comment:
This is related to the comments below about `new` vs `new_color_object`, but I think we might just want to give the module an idiomatic name (`ColorObject`) and indicate the association with chapter 1 a different way (subfolder in `sources/` perhaps?).